---
float32 goals_reached
float32 goals_cancelled